packages/google_maps_flutter/google_maps_flutter/example/test_driver/google_maps_e2e.dart

```diff
@@ -823,4 +824,31 @@ void main() {
     iwVisibleStatus = await controller.isMarkerInfoWindowShown(marker.markerId);
     expect(iwVisibleStatus, false);
   });
+
+  testWidgets('testTakeSnapshot', (WidgetTester tester) async {
+    Completer<GoogleMapInspector> inspectorCompleter =
+        Completer<GoogleMapInspector>();
+
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: GoogleMap(
+          initialCameraPosition: _kInitialCameraPosition,
+          onMapCreated: (GoogleMapController controller) {
+            final GoogleMapInspector inspector =
+                // ignore: invalid_use_of_visible_for_testing_member
+                GoogleMapInspector(controller.channel);
+            inspectorCompleter.complete(inspector);
+          },
+        ),
+      ),
+    );
+
+    await tester.pumpAndSettle(const Duration(seconds: 3));
+
+    final GoogleMapInspector inspector = await inspectorCompleter.future;
+    final Uint8List bytes = await inspector.takeSnapshot();
+    expect(bytes, isNotNull);
```

Comment:
we can probably remove this as if `bytes` is null, the below test would fail as well.